addons/mail/static/src/models/thread_needaction_preview_view.js

```diff
@@ -2,11 +2,32 @@
 
 import { registerModel } from '@mail/model/model_core';
 import { one } from '@mail/model/model_field';
+import { clear, insertAndReplace } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'ThreadNeedactionPreviewView',
     identifyingFields: ['notificationListViewOwner', 'thread'],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeMessageAuthorPrefixView() {
+            if (
+                this.thread.lastNeedactionMessageAsOriginThread &&
+                this.thread.lastNeedactionMessageAsOriginThread.author
+            ) {
+                return insertAndReplace();
+            }
+            return clear();
+        },
```

Comment:
There is multiple pattern to create view, I have seen some using ternary: 

```javascript
        _computeMessageView() {
            return this.message ? insertAndReplace({ message: replace(this.message) }) : clear();
        },
```
For simple view creation, maybe one line return is a better option ?
We could also find something better than compute every view ? It's becoming a recurring pattern.

Reply:
I'm not fond of ternary. I'm okay using them when the condition is small, but here it's a long expression, so definitely not use ternary.

Reply:
The pattern applies to any relational field that is declarative enough, so it's not just with views.
I don't want to artificially make it more succinct.
Right now we need the flexibility of algorithmically compute the field, so we can't just reduce noise with a rudimentary Domain-Specific-Language (DSL) kind of declaration of how it's computed.